gh-cli-research/03-timeline-and-metrics.md

```
@@ -0,0 +1,1194 @@
+    }
+  ' --jq '.data.repository.pullRequest.closingIssuesReferences.nodes'
+```
+
+### 6.2 From Issue to PRs (closedByPullRequestsReferences -- GraphQL only)
+
+There is **no REST API equivalent**. You must use GraphQL:
+
+```bash
+gh api graphql -F owner='raulstechtips' -F repo='calendar-agent' -F issue=100 \
+  -f query='
+    query($owner: String!, $repo: String!, $issue: Int!) {
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)